examples/basic/add/run.py

```diff
@@ -4,11 +4,15 @@
 from oremda import Operator as OremdaOperator
 
 class AddOperator(OremdaOperator):
-    def kernel(self, input_data, parameters):
+    def kernel(self, meta, data, parameters):
         value = parameters.get('value', 0)
-        output_data = input_data + value
+        input_data = data.get('scalars')
 
-        return output_data
+        output = {
+            'scalars': input_data + value
+        }
+
+        return meta, output
```

Comment:
In case multiple data objects get returned, should we consider returning a dict instead, to keep track of which data is which, like:
```python
return {
    "meta": meta,
    "output": output,
}
```

Reply:
It's already like this, if you look at that code what is returned is the following:
```python
return (
    { # The various meta objects
    },
    { # The various data arrays
        'scalars': input_data + value
    }
)
```

Reply:
Ah I see, thank you!